src/style.css

```diff
@@ -1 +1,57 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.logo__img {
+  margin: 10px 0 10px 50px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+  flex-direction: row;
+}
+
+.nav__link {
+  line-height: 60px;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  margin-right: 20px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.nav__link:last-child {
+  margin-right: 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.products {
+  display: flex;
+}
+
+.is-active::after {
+  position: absolute;
+  top: 56px;
```

Comment:
Please use bottom: 0; because in this case will more correctly position the element at the bottom of the parent